Wrangler is broken

### Describe the bug
Wrangler is broken on version 3.0.1

### Steps To Reproduce
nix shell nixpkgs#wrangler (latest flake)
wrangler dev
--> Error: spawn /nix/store/d70h3plhhzpx0wrn20dpyghi3nrp28wx-wrangler-3.0.1/lib/node_modules/wrangler/node_modules/@cloudflare/workerd-linux-64/bin/workerd ENOENT

### Expected behavior
wrangler dev should start the server

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.1-zen1, NixOS, 23.11 (Tapir), 23.11.20230529.5e871d8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(chebuya): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

greetings !

I am not familiar enough with wrangler, but maybe the issue has been fixed ?

```
 nix shell nixpkgs#wrangler

[kirillvr@tsutenkaku:~]$ wrangler dev
 ⛅️ wrangler 3.1.0
------------------
wrangler dev now uses local mode by default, powered by 🔥 Miniflare and 👷 workerd.
To run an edge preview session for your Worker, use wrangler dev --remote

✘ [ERROR] Missing entry-point: The entry-point should be specified via the command line (e.g. `wrangler dev path/to/script`) or the `main` config field.


If you think this is a bug then please create an issue at https://github.com/cloudflare/workers-sdk/issues/new/choose

```

--- a/pkgs/development/node-packages/overrides.nix
+++ b/pkgs/development/node-packages/overrides.nix
@@ -597,6 +597,11 @@ final: prev: {
 
   wrangler = prev.wrangler.override (oldAttrs: {
     meta = oldAttrs.meta // { broken = before "16.13"; };
+    buildInputs = [ pkgs.llvmPackages.libcxx pkgs.llvmPackages.libunwind ] ++ lib.optional stdenv.isLinux pkgs.autoPatchelfHook;
+    preFixup = ''
+      # patch elf is trying to patch binary for sunos
+      rm -r $out/lib/node_modules/wrangler/node_modules/@esbuild/sunos-x64
+    '';
   });
 
   yaml-language-server = prev.yaml-language-server.override {

